opbeans/src/main/java/co/elastic/apm/opbeans/controllers/IndexController.java

```diff
@@ -39,8 +40,15 @@ public String redirect() {
     }
 
     @RequestMapping({ "/is-it-coffee-time" })
-    public String error() {
-        throw new RuntimeException("Demo exception");
+    public String error(@RequestParam(value = "cause", required = false) String causeMsg) {
```

Comment:
It'd just make it throw an exception with a cause unconditionally. The load generator for the demo environment does not know about the `cause` parameter and it's easier to get test data of chained exceptions if you don't have to manually execute the endpoint and remember the parameter name.

Reply:
That's fair, I've removed this part.